tensorflow/core/kernels/data/experimental/snapshot_dataset_op.cc

```diff
@@ -83,6 +83,8 @@ namespace experimental {
     SnapshotDatasetV2Op::kReaderFuncTarguments;
 /* static */ constexpr const char* const
     SnapshotDatasetV2Op::kShardFuncTarguments;
+/* static */ constexpr const char* const SnapshotDatasetV2Op::kReaderPrefix;
```

Comment:
these are already defined on L71 and L72

Reply:
Oh, I'm sorry. I fixed this in the 2.4.1 release, where it was missing, and didn't see that it is already fixed in master.